.scripts/CreateNuget.ps1

```diff
@@ -0,0 +1,201 @@
+[CmdletBinding()]
```

Comment:
This is the main script that would be called on CI. It should be called like below:
`CreateNuget.ps1 -msbuild "path_to_msbuild\MSBuild.exe" -nuget "path_to_nuget\NuGet.exe" -outputDir path_to_output_directory -ApiKey "API_KEY_FROM_NUGET_ORG"`

I'm not sure how the CI works, so for now the ApiKey should be passed as a string, but perhaps it should be passed as credentials or securestring?